mkdocs.yml

```diff
@@ -99,8 +106,6 @@ theme:
     - navigation.top
     - navigation.tracking
     - toc.follow
-    - versioning:
-        provider: mike
```

Comment:
I think this will need to stay for the versioning to continue to work. Unless we want to switch to a different system for managing documentation versions?